docker/setup.sh

```diff
@@ -62,33 +83,80 @@ update_ini_values
 if [[ ! -d "/client" ]]; then
     echo "Client not found."
     echo "Did you forget to mount the client into the \"/client\" directory?"
-    exit 1
+    exit 12
 fi
 
-if [[ ! -f "/client/extracted" ]]; then
+# Try to auto detect client type
+if [[ "${CLIENT_TYPE@L}" == "auto" ]]; then
+    if [[ -f "/client/legouniverse.exe" ]]; then
+        # Look for a unpacked file. If it doesn't exist, then we can assume the user provided a packed client
+        # with a client structure like a unpacked client. At this point we can't continue the process. 
+        # Without the versions directory lunpack can't extract the client files
+        if [[ ! -f "/client/res/maps/nexustower/nd_nexus_tower_7x1_audio.lvl" ]]; then
```

Comment:
I would use cdclient.fdb not a map file that might be changed by future content.

Reply:
I picked this particular file because I know that there will be issues when this file is missing. But, yeah, maybe cdclient.fdb is a better choice.

Reply:
> I picked this particular file because I know that there will be issues when this file is missing
If cdclient.fdb is missing and the client is running unpacked then it will crash on start, it has to have this file where as it doesn't need some audio scene from NT

Reply:
> > I picked this particular file because I know that there will be issues when this file is missing
> 
> If cdclient.fdb is missing and the client is running unpacked then it will crash on start, it has to have this file where as it doesn't need some audio scene from NT

The client won't join the server if this file is missing (client missmatch, if I remember correctly). I will add `cdclient.fdb` as check.